README.md

```diff
@@ -1,24 +1,132 @@
+- [Prerequisites](#prerequisites)
+- [Setup](#setup)
+- [Usage](#usage)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* System dependencies
+# 📖 [HELLO-RAILS-REACT] <a name="about-project"></a>
 
-* Configuration
+**HELLO-RAILS-REACT** is a greeting app that is built rails and react together using webpack and babel as a bundler.
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+    <li>React</li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- **Create the Greeting component that will display a greeting**
+- **Create an API endpoint that selects a random greeting from the database table**
+- **Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting**
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+to install git on your computer. Follow the instruction on the page to do so
+[Click here ](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the instruction on the page to install git.
+[Click here ](https://www.ruby-lang.org/en/documentation/installation/) and follow the instruction on the page to install ruby.
+[Click here ](https://guides.rubyonrails.org/v5.1/getting_started.html) and follow the instruction on the page to install rails.
+[Click here ](https://www.postgresql.org/download/) and follow the instruction on the page to install postgres.
+
+### Setup
+
+Open your preferred folder in the terminal
+
+Clone this repository to your desired folder:
+Using the commands below:
+
+```sh
+  git clone https://github.com/emarkees/hello-rails-react.git
+```
+
+Open the project on your code editor
+
+Change directory to the project folder
+
+```sh
+  cd hello-rails-react
+```
+
+### Usage
+
+Then, run the following command to start the app
+
+```sh
+    ./bin/dev
+```
+
+## 👥 Author <a name="author"></a>
+
+👤 **Adeyemi Olusola**
+
+- GitHub: [@Adeyemi-arobieke](https://github.com/emarkees)
+- Linkedin: [@Adeyemi](https://linkedin.com/in/adeyemi-olusola)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add calendar to the app**
+- [ ] **Display greeting based on time, day, and holiday**
+- [ ] **Save greeting message to the database**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+- [Coming Soon]
```

Comment:
- [Optional] I think you forgot to remove this line. Let's kindly remove the line in order to render readme file professional :+1: